src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
```

Comment:
The function name 'calculateRentalCost' is descriptive and clearly states its purpose.